### ISSUE TITLE
zc156: Attributes Controller, can't add new attribute

### ISSUE THREAD

**Issue by lat9** [zencart/zencart#1673]
With an install of the demo products, I created a copy of the _**Microsoft Internet Keyboard**_ product and went to the Attributes Controller to add some attributes for testing.

I selected the "Color" option, the "Black" color and pressed the `Insert` button.  Upon click, the screen returned to the main Attributes Controller display with the "Please select a category ..." message.

Looking in the /logs directory, I found the following log:
```
[27-Apr-2018 10:36:07 America/New_York] PHP Notice:  Undefined offset: 2 in C:\xampp\htdocs\zc156\admin\includes\modules\products_previous_next.php on line 95
[27-Apr-2018 10:36:07 America/New_York] PHP Notice:  Undefined index: products_id in C:\xampp\htdocs\zc156\admin\attributes_controller.php on line 252
[27-Apr-2018 10:36:07 America/New_York] PHP Notice:  Undefined index: products_id in C:\xampp\htdocs\zc156\admin\attributes_controller.php on line 263
[27-Apr-2018 10:36:07 America/New_York] PHP Notice:  Undefined index: products_id in C:\xampp\htdocs\zc156\admin\attributes_controller.php on line 379
[27-Apr-2018 10:36:07 America/New_York] PHP Notice:  Undefined index: products_id in C:\xampp\htdocs\zc156\admin\attributes_controller.php on line 381
[27-Apr-2018 10:36:07 America/New_York] PHP Notice:  Undefined index: current_category_id in C:\xampp\htdocs\zc156\admin\attributes_controller.php on line 381
```

On inspection of the database, a `products_attributes` record was added ... with a `products_id` of **0**.

**Comment by lat9** [comment]
Yes, unfortunately that error "note" was made to an issue that was closed around 6 months ago; that's why I didn't see it!

**Comment by Zen4All** [comment]
You can't keep track of everything

**Comment by Zen4All** [comment]
I'll make a pr with the fix this week

**Comment by drbyte** [comment]
Closed by #1720
